src/app/dashboard/main-boards/main-boards.component.html

```diff
@@ -28,7 +29,8 @@ <h4 class="main-board-title" (click)="toggleTitleEditMode()">{{ boardTitle }}</h
 <ng-template #editDescription>
   <input
     #inputDescription
-    class="edit-input main-board-input"
+    matInput
+    class="main-board-input mt-5 mb-5"
```

Comment:
can be improved: if we need the same value for margin we can add a single class in utils for it